sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/client.py

```diff
@@ -56,75 +56,75 @@
+    :paramtype properties: dict
+    :keyword remote_idle_timeout_empty_frame_send_ratio: Ratio of empty frames to
      idle time for Connections with no activity. Value must be between
      0.0 and 1.0 inclusive. Default is 0.5.
-    :type remote_idle_timeout_empty_frame_send_ratio: float
-    :param incoming_window: The size of the allowed window for incoming messages.
-    :type incoming_window: int
-    :param outgoing_window: The size of the allowed window for outgoing messages.
-    :type outgoing_window: int
-    :param handle_max: The maximum number of concurrent link handles.
-    :type handle_max: int
-    :param on_attach: A callback function to be run on receipt of an ATTACH frame.
+    :paramtype remote_idle_timeout_empty_frame_send_ratio: float
+    :keyword incoming_window: The size of the allowed window for incoming messages.
+    :paramtype incoming_window: int
+    :keyword outgoing_window: The size of the allowed window for outgoing messages.
+    :paramtype outgoing_window: int
+    :keyword handle_max: The maximum number of concurrent link handles.
+    :paramtype handle_max: int
+    :keyword on_attach: A callback function to be run on receipt of an ATTACH frame.
      The function must take 4 arguments: source, target, properties and error.
-    :type on_attach: func[~uamqp.address.Source, ~uamqp.address.Target, dict, ~uamqp.errors.AMQPConnectionError]
-    :param send_settle_mode: The mode by which to settle message send
+    :paramtype on_attach: func[~pyamqp.endpoint.Source, ~pyamqp.endpoint.Target, dict, ~pyamqp.error.AMQPConnectionError]
+    :keyword send_settle_mode: The mode by which to settle message send
      operations. If set to `Unsettled`, the client will wait for a confirmation
      from the service that the message was successfully sent. If set to 'Settled',
      the client will not wait for confirmation and assume success.
-    :type send_settle_mode: ~uamqp.constants.SenderSettleMode
-    :param receive_settle_mode: The mode by which to settle message receive
+    :paramtype send_settle_mode: ~pyamqp.constants.SenderSettleMode
+    :keyword receive_settle_mode: The mode by which to settle message receive
      operations. If set to `PeekLock`, the receiver will lock a message once received until
      the client accepts or rejects the message. If set to `ReceiveAndDelete`, the service
      will assume successful receipt of the message and clear it from the queue. The
      default is `PeekLock`.
-    :type receive_settle_mode: ~uamqp.constants.ReceiverSettleMode
-    :param encoding: The encoding to use for parameters supplied as strings.
+    :paramtype receive_settle_mode: ~pyamqp.constants.ReceiverSettleMode
+    :keyword encoding: The encoding to use for parameters supplied as strings.
      Default is 'UTF-8'
-    :type encoding: str
+    :paramtype encoding: str
     """
 
-    def __init__(self, hostname, auth=None, **kwargs):
+    def __init__(self, hostname, **kwargs):
```

Comment:
Are we also making auth a keyword -- will need to do the same on the Send and Receive Clients